concepticondata/conceptlists/Teit-1908-693.tsv

```diff
@@ -0,0 +1,694 @@
+Teit-1908-693-154	154		abdomen			50
+Teit-1908-693-155	155	137	flesh & meat	2615	FLESH OR MEAT	50
+Teit-1908-693-156	156	138	bone	1394	BONE	52
+Teit-1908-693-157	157	139	sinew	1137	TENDON	52
+Teit-1908-693-158	158	140	footprint	318	FOOTPRINT	52
+Teit-1908-693-159	159	141	brain	1583	BRAIN	52
+Teit-1908-693-160	160	142	myths			52
+Teit-1908-693-161	161	143	ghost	1175	GHOST	52
+Teit-1908-693-162	162	144	shadow	2275	SHADOW	52
+Teit-1908-693-163	163	145	soul	811	SOUL	52
+Teit-1908-693-164	164	146	sick	1847	SICK	52
+Teit-1908-693-165	165	147	corpse	767	CORPSE	52
+Teit-1908-693-166	166	148	dead	1625	DEAD	52
+Teit-1908-693-167	167	149	grave	1773	GRAVE	52
+Teit-1908-693-168	168	150	hat (any kind)	771	HAT	52
+Teit-1908-693-169	169	151	headband	330	HEADBAND	52
+Teit-1908-693-170	170	152	feather headdress			52
+Teit-1908-693-171	171		feather bonnet			52
+Teit-1908-693-172	172	153	bunch of feathers on head or band			52
+Teit-1908-693-173	173	154	necklace	1961	NECKLACE	52
+Teit-1908-693-174	174		around throat			52
+Teit-1908-693-175	175	155	neckerchief			52
+Teit-1908-693-176	176	156	handkerchief	1020	HANDKERCHIEF	52
+Teit-1908-693-177	177	157	dressed skin shirt			52
+Teit-1908-693-178	178	158	[dressed skin shirt] man			54
+Teit-1908-693-179	179	159	[dressed skin shirt] woman			54
+Teit-1908-693-180	180	160	apron (any kind)	2046	APRON	54
+Teit-1908-693-181	181	161	apron (old style)			54
+Teit-1908-693-182	182	162	breech clout	1965	LOINCLOTH	54
+Teit-1908-693-183	183	163	long leggings			54
+Teit-1908-693-184	184	164	short leggings			54
+Teit-1908-693-185	185	165	moccasins (general name)	3607	MOCCASIN	54
+Teit-1908-693-186	186	166	blanket [moccasins]			54
+Teit-1908-693-187	187	167	buffalo skin			54
+Teit-1908-693-188	188	168	buffalo robe			54
+Teit-1908-693-189	189	169	deer skin			54
+Teit-1908-693-190	190	170	elk [skin]			54
+Teit-1908-693-191	191	171	dressed bucksin (any kind)			54
+Teit-1908-693-192	192	172	rabbit skin robe			54
+Teit-1908-693-193	193		jackrabbit [skin robe]			54
+Teit-1908-693-194	194		small rabbit [skin robe]			54
+Teit-1908-693-195	195	173	cape	1662	CAPE	54
+Teit-1908-693-196	196	174	poncho	327	PONCHO	54
+Teit-1908-693-197	197	175	thread (in general)	1161	THREAD	54
+Teit-1908-693-198	198	176	sinew thread			54
+Teit-1908-693-199	199	177	awl (for sewing)	278	AWL	54
+Teit-1908-693-200	200	178	thimble			54
+Teit-1908-693-201	201	179	skin scraper			54
+Teit-1908-693-202	202	180	skin dresser			54
+Teit-1908-693-203	203		stone head			54
+Teit-1908-693-204	204	181	chisel	1979	CHISEL	54
+Teit-1908-693-205	205	182	hammer of [chisel]			56
+Teit-1908-693-206	206	183	mortar	224	MORTAR	56
+Teit-1908-693-207	207	184	pestle	225	PESTLE	56
+Teit-1908-693-208	208	185	fire-drill	517	FIREDRILL	56
+Teit-1908-693-209	209	186	axe	677	AXE	56
+Teit-1908-693-210	210	187	knife	1352	KNIFE	56
+Teit-1908-693-211	211	188	knife-handle			56
+Teit-1908-693-212	212	189	cup	476	CUP	56
+Teit-1908-693-213	213	190	plate	480	PLATE	56
+Teit-1908-693-214	214		mat			56
```

Comment:
195	MAT ?

Reply:
Sorry, I missed this gloss that should've been restored to _[plate] mat_. The glosses with no number in the column NUMBER _IN_SOURCE are usually related to the main entry. Here _mat_ should refer to _plate_ above (they are listed under the same number in the source). I assumed that it's some kind of a mat for a plate.